README.md

```diff
@@ -1,14 +1,13 @@
-
-## Microsoft Open Source Code of Conduct
-
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
-For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
-
 # Visual F# Documentation
 
 [![Project Status: Suspended - Initial development has started, but there has not yet been a stable, usable release; work has been stopped for the time being but the author(s) intend on resuming work.](http://www.repostatus.org/badges/latest/suspended.svg)](http://www.repostatus.org/#suspended)
```

Comment:
This is the official reference documentation for FSharp.Core.  All the conceptual content, including the language reference, is moved over to docs.microsoft.com.  We're still taking contributions here, but only for updating FSharp.Core reference material.